OrbitGl/App.cpp

```diff
@@ -306,6 +304,21 @@ void OrbitApp::PostInit() {
       ERROR("Unable to create GRPC channel to %s",
             options_.grpc_server_address);
     }
+
+    // TODO: Replace refresh_timeout with config option. Let users to modify it.
+    process_list_manager_ =
+        ProcessListManager::Create(grpc_channel_, absl::Milliseconds(1000));
+
+    auto callback = [&](std::vector<ProcessInfo>&& process_list) {
+      main_thread_executor_->Schedule(
+          [&, list = std::move(process_list)]() mutable {
```

Comment:
This is a tad obscure, maybe because I had never seen it. Are we sure there is no better way than moving into a captured variable and using `mutable`?

Reply:
I have not found any.. I could change it so it is const reference which would imply some extra copies... probably 2.

Reply:
Yeah, while copies are not a problem, they're not elegant as they can be avoided.
Another way would be to have a custom `Action` in which you can store a process list, which would basically do the same as the lambda but with more explicit code, but maybe it's not worth it as it would inflate the code to do the exact same thing.

Reply:
Actually, shared_ptr is another way to go. unique_ptr will still require mutable and move but shared_ptr should be fine. This will however mask ownership of the vector which is de-facto ProcessesDataView

Reply:
Mmh, I don't know if I like messing with ownership so much better.

Reply:
leaving this as is for a now.